docs/search/get-started-with-search/search-page/navigate-messages-in-search-results.md

```diff
@@ -6,54 +6,53 @@ description: When you run a search query, messages display in the Message, Aggre
+The **Signatures** tab is shown when using [LogReduce](/docs/search/logreduce).
 
 ![Signatures tab.png](/img/search/get-started-search/search-page/signatures-tab.png)
 
-## Column adjustments
+## Column Adjustments
 
-The table columns can be modified by the following:
+The table columns can be modified by the following:
 
-* **Move a Column.** Hold click and drag a column from the header row to a different location in the table.
-* **Change column width.** Hold click and drag the vertical line in between the columns to adjust the width. <br/>![resize column.png](/img/search/get-started-search/search-page/resize-column.png)
-* **Change column height.** Double click the vertical line to the right of a column name to automatically resize the width to fit the content.
-* **Pin columns**: You can pin columns in the table, either in the **Messages** tab or the **Aggregates** tab. When a column is pinned you can scroll to the right and still view it. To pin a column, hover over the column header, and click the pin icon that appears.
+* **Move a Column.** To move a column, click and hold down your mouse button on the column, then drag it to a new location.
+* **Change column width**. To adjust the column width, click and hold the vertical line separating the columns, and drag it left or right. <br/>![resize column.png](/img/search/get-started-search/search-page/resize-column.png)
+* **Change column height.** Double click the vertical line to the right of a column name to automatically resize the width to fit the content.
+* **Pin columns**: You can pin columns in the table, either in the **Messages** tab or the **Aggregates** tab. When a column is pinned, you can scroll to the right and still view it. To pin a column, hover over the column header, and click the pin icon that appears.
 
 After you’ve pinned a column, the pin icon has a slash through it. You can click the icon to unpin the column.
 
 
 
 
-## Expand and collapse
+## Expand and Collapse
 
-By default, the **Messages** tab shows the collapsed version of messages, where the row has a height limit of 500 pixels. Click **Expand All** on the upper right side of the **Messages** table to view the full contents of all your log messages.
+By default, the **Messages** tab shows the collapsed version of messages with each row showing up to 10 lines. To see the full contents of all log messages, including JSON logs, click the **Expand/Collapse** button on the upper right side of the **Messages** table and choose either **Expand All Rows** or **Expand All Rows And JSON** option.
 
 ![expand message option.png](/img/search/get-started-search/search-page/expand-message-option.png)
 
-If expanded, the option changes to **Collapse All**. Click it to switch back the initial collapsed view.
+Click  **Expand/Collapse** on the upper right side of the **Messages** table and select  **Collapse All Rows**  to switch back to the initial collapsed view.
 
-## Copy message
+## Copy Message
```

Comment:
Reverting edit. Our style is to use Sentence Case (see [Style Guide](https://help.sumologic.com/docs/contributing/style-guide/#capitalization)).

```suggestion
## Copy message
```